### DISCUSSION TITLE
How can I set it up to search for a 3070ti? I don't see it that card as an option.

### DISCUSSION THREAD

**Discussion by Kafei56** [jef/streetmerchant#2756] [Help]
I'd like to also scan for a 3070ti but I'm not seeing how, is that card omitted for a reason?

**Comment by Mr-Tech-13** [comment]
No, its not added. I can send you a google drive link to my 3070ti bot if you would like

**Reply by MoncefAfif** [reply]
can you show us or refer us to a certain page that shows how to add other products

**Reply by tv1er** [reply]
Can you send it to me?

**Reply by honeymonkey35** [reply]
ı need this

**Reply by Mr-Tech-13** [reply]
https://drive.google.com/drive/folders/1tK_OeoClXe9LjFm-XbI8ip6VToy7Jm28?usp=sharing

This might be a little out of date, so please tell me if it's missing anything...
